src/pages/components/map-related/MapContent.tsx

```diff
@@ -0,0 +1,93 @@
+import { Marker, Polyline, Popup, TileLayer } from 'react-leaflet'
+import { Icon, IconOptions } from 'leaflet'
+import { useAgencyList } from 'src/api/agencyList'
+import { busIcon, busIconPath } from '../utils/BusIcon'
+import { BusToolTip } from './MapLayers/BusToolTip'
+import { t } from 'i18next'
+import '../../Map.scss'
+import { MapIndex, MapProps } from './MapWithLocationsAndPath'
```

Comment:
you can create a different file for this component and it will probably solve the circular dependency problem